### DISCUSSION TITLE
Channel Open transaction is stuck

### DISCUSSION THREAD

**Discussion by ManuAgora** [Ride-The-Lightning/RTL#1257] [Troubleshooting Questions] [answered]
Hi all, first the basics. I am running:
RTL v0.13.6~1
on CLN 23.02.2~3
on Start9 OS 0.3.4~2

My issue:
(A long time ago) I tried opening a channel with a minimal sat/vB option and RTL said this would be ignored by peers. So I immediately tried another channel opening with the same utxo with a higher (15 sat/vB) fee rate. Fees were much higher for weeks but recently dropped well below 15. I thought my channel opening was going to be complete but now on the RTL peers/channels, it says that the channel is still not opened and both of these transaction IDs don't exist in mempool(dot)space. 
Please can you help me. I'm trying to broadcast one of those (since they are the same utxo) so that I can open the channel. I used the whole UTXO for opening the channel and paying miner fee, since that is best practice for privacy, but now i cannot bump the fee with child pays for parent (because i have no change output). Ideally I could re-broadcast the transaction since it is purged from mempool(dot)space. No other mempools must have my transaction, otherwise I am sure i would se the transaction, even if unconfirmed.

**Comment by saubyk** [comment, marked as answer]
Hi @ManuAgora 

Given your above description, this is what seem to have transpired:
1. You opened a channel with a fee rate of 15sats/vB, which was less than the prevailing rate at the time
2. Your transaction was either, not accepted in the mempool, or dropped out after two weeks (if it was)
3. Your node has locked the UTXO for 2000 blocks (which is a usual practice on core lightning before attempting channel open)
4. Now you're not able to view the UTXO because it's locked

Please note that you will **not** be able to fee bump this channel open transaction at all, since it's no where to be found in the mempool
Here's what you can do.

Option # 1
If you know how to operate command line. Open a terminal on your node and run this core lightning command `lightning-cli listfunds`. This command output will show you any locked UTXOs on your node and how many blocks it's still locked for.

Option # 2
If you don't know how to operate command line interface, just wait for a few more days until it's been 2000 blocks since you attempted channel open and your UTXO shall reappear in the UI.

For future reference, **always** check mempool before submitting onchain transactions and set competitive rates, so that you don't end up in similar situation again. It's a bidding system and only transactions with competitive rates make it into blocks, regardless of how much we want to pay.

Hope this helps.
